src/validation/isValidGroupOptions.ts

```diff
@@ -20,7 +20,7 @@ export function isValidGroupOptions(root: string, groups: VersionGroupOptions[])
   for (const grp of Object.keys(packageGroups)) {
     const pkgs = packageGroups[grp].packageNames;
     for (const pkgName of pkgs) {
-      if (packageInfos[pkgName].options.disallowedChangeTypes) {
+      if (packageInfos[pkgName].options.disallowedChangeTypes && !packageGroups[grp].disallowedChangeTypes) {
```

Comment:
Not quite understanding what's going on here, sorry--would you mind explaining a bit more?

Reply:
@kenotron added this validation originally, so please correct me if i'm wrong here. 

My understanding is we want users to define `disallowedChangeTypes ` for the entire group rather than allowing each package to define individually. otherwise beachball doesn't know how to behave if packages within the group have different `disallowedChangeTypes`.

Reply:
The intention is to disallow any "disallowedChangeTypes" settings within a package for any packages defined in a group. This is regardless of whether you have a setting for the group or not. I'm not sure what bug you're addressing. Would like to see an example here.